Doc/library/bdb.rst

```diff
@@ -311,16 +360,18 @@ The :mod:`bdb` module also defines two classes:
 
    .. method:: get_stack(f, t)
 
-      Get a list of records for a frame and all higher (calling) and lower
-      frames, and the size of the higher part.
+      Return a list of (frame, lineno) tuples in a stack trace, and a size.
+
+      List starts with the original calling frame, if there is one.  Size may
+      be the number of frames above or below *f*.
```

Comment:
This is incorrect, the original frame is somewhere in the middle. From reading the code, I think this is right:

```suggestion
      The most recently called frame is last in the list. The size is the number
      of frames below the frame where the debugger was invoked.
```